Cargo.toml

```diff
@@ -6,5 +6,7 @@ members = [
     "images/trufflesuite_ganachecli",
     "images/coblox_bitcoincore",
     "images/parity_parity",
-    "images/dynamodb_local"
+    "images/dynamodb_local",
```

Comment:
🙃 I know it was not already sorted but could you sort the lines here please?

Reply:
Done!